utils/data-migration/remove-sql-ledger/copy-and-remove-ledger.feature.sql

```diff
@@ -0,0 +1,360 @@
+-- ENABLE ROLLBACK TRANSACTION ON ERROR
```

Comment:
I think I'd prefer to reset feature and run migrations, as that's closer to what will happen on staging + prod (minimal changes to a migration that exists then a new migration). Happy to leave this in for now anyway